nav2_bashrc

```diff
@@ -0,0 +1,14 @@
+OVERLAY_DIR=/opt/overlay_ws
```

Comment:
This should be in `/tools` not in the root of the repository please

Reply:
Ah nice, I was looking for a better place

Reply:
`/tools` is our catch-all for random developer tools